src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const inverse = {};
+
+  for (const key in robot) {
+    if (`${robot[key]}` in inverse) {
```

Comment:
consider using `Object.hasOwnProperty()` method for checkling if our new value is in object